### ISSUE TITLE
[Bug] Light mode does not persist on title page after refresh

### ISSUE THREAD

**Issue by travisk-codes** [the-programmers-hangout/website#334]
**Description:**
When light mode is enabled, refreshing the title page will revert it back to dark mode.

**To Reproduce:**
1. Enable light mode
2. Navigate back to the title page
3. Refresh the page

**Expected Behavior:**
Light mode to persist after refresh

**Environment:**
- OS: 5.8.6-arch1-1
- Browser: Firefox 80.0.1 (64-bit)
- Browser: Chromium 85.0.4183.83 (64-bit)

**Comment by veksen** [comment]
The issue seems to happen on every layout, part of the layout doesn't take account the localStorage value:

https://theprogrammershangout.com/resources/javascript/

![image](https://user-images.githubusercontent.com/3461986/92468622-74cf6900-f1a1-11ea-8d7f-145cb3603ba3.png)

**Comment by travisk-codes** [comment]
I am able to reproduce on prod, but _not_ on develop `master`. Are you?

**Comment by veksen** [comment]
I'm feeling like it's related to statically generated generated Gatsby assets

**Comment by veksen** [comment]
yes 100%: https://joshwcomeau.com/gatsby/dark-mode/ improved solution from what we have

**Comment by travisk-codes** [comment]
I'll look into it--does it look like a major overhaul?

**Comment by veksen** [comment]
No we have the majority of the groundwork already for it, it's mainly injecting some JS at runtime before Gatsby runs which is covered by the article.
